cli/tests/error_syntax_empty_trailing_line.mjs.out

```diff
@@ -1,2 +1 @@
-error: Uncaught SyntaxError: Unexpected end of input
-    at [WILDCARD]tests/error_syntax_empty_trailing_line.mjs:[WILDCARD]
+error: Unexpected eof at [WILDCARD]tests/error_syntax_empty_trailing_line.mjs:2:21
```

Comment:
I think the same here... "eof" is sometimes well understood, but "Unexpected end of file" is clearer.  No sense in holding up this PR, but maybe we should have a DX issue for error messages for this.  Hopefully something the wider community would help out with.